src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,90 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
+
+.card {
+    width: 100%;
+    margin: 10px auto;
+    background-color: white;
+    box-shadow: 0px 5px 20px #555;
+    border: none;
+}
+
+.card .card-header {
+    position: relative;
+    padding: .75rem 1.25rem;
+    margin-bottom: 0;
+    background-color: rgba(0,0,0,0.03);
+    border-bottom: 1px solid rgba(0,0,0,0.125);
+}
+.card .card-header h3{
+    color: #fff;
+    font-weight: 400;
+}
+.card a {
+    color: #333;
+    text-decoration: none;
+}
+
+.card:hover .card-image img {
+    width: 110%;
+    filter: grayscale(0);
+}
+
+.card-image {
+    position: relative;
+    overflow: hidden;
+}
+
+.card-image img {
+    width: 100%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    filter: grayscale(0.2);
+    transition-property: filter width;
+    transition-duration: .3s;
+}
+
+.card-body {
+    /*text-align: center;*/
+    padding: 15px 20px;
+    box-sizing: border-box;
+}
+
```

Comment:
I am pretty unsure about the following generic card styles and fonts in general.